src/mofdscribe/featurizers/topology/ph_hist.py

```diff
@@ -34,14 +34,14 @@ class PHHist(MOFBaseFeaturizer):
 
     def __init__(
         self,
-        atom_types: Tuple[str] = (
+        atom_types: List[str] = [
```

Comment:
Tuple was originally used because it is immutable and hence a better default for arguments. 

But since we do not change the `atom_types` this is probably a safe choice